### DISCUSSION TITLE
Error with MySQL DateTime! str type expected (type=type_error.str)

### DISCUSSION THREAD

**Discussion by AliMuhammadOfficial** [tiangolo/pydantic-sqlalchemy#98] [Questions] [answered]
I have created my model from pre-existing MySQL database everything working fine butt when I add DateTime: str gives error. str type expected (type=type_error.str)

**Comment by ljluestc** [comment]
from pydantic import BaseModel
from datetime import datetime

class YourPydanticModel(BaseModel):
    datetime_column: datetime  # Use appropriate datetime type here

# Example usage
data = {
    "datetime_column": "2023-09-03T12:34:56"  # Replace with your actual datetime string
}

pydantic_instance = YourPydanticModel(**data)
print(pydantic_instance)

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! :sweat_smile:

This would be a feature request.

I deprecated this library a while ago in favor of SQLModel which would solve all the same use cases in a much better way, and I'm directing all the effort in that direction.

SQLModel uses Pydantic models to define both data and table (SQL) models, so it's easier to do all these things. :rocket:
